lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -0,0 +1,155 @@
+require 'fastlane_core/ui/ui'
+require 'fastlane/action'
+require 'fastlane/actions/github_api'
+require 'fastlane/actions/push_to_git_remote'
+require 'fastlane/actions/create_pull_request'
+require 'fastlane/actions/ensure_git_branch'
+require 'fastlane/actions/ensure_git_status_clean'
+
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?(:UI)
+
+  module Helper
+    class RevenuecatInternalHelper
+      def self.replace_version_number(previous_version_number, new_version_number, files_to_update, files_to_update_without_prerelease_modifiers)
+        previous_version_number_without_prerelease_modifiers = previous_version_number.split("-")[0]
+        new_version_number_without_prerelease_modifiers = new_version_number.split("-")[0]
+
+        files_to_update.each do |file_to_update|
+          replace_in(previous_version_number, new_version_number, file_to_update)
+        end
+        files_to_update_without_prerelease_modifiers.each do |file_to_update|
+          replace_in(previous_version_number_without_prerelease_modifiers, new_version_number_without_prerelease_modifiers, file_to_update)
+        end
+      end
+
+      def self.auto_generate_changelog(repo_name, github_token, rate_limit_sleep)
+        old_version = Actions.sh("git describe --tags --abbrev=0").strip
+        UI.important("Auto-generating changelog since #{old_version}")
+
+        org = "RevenueCat"
+
+        path = "/repos/#{org}/#{repo_name}/compare/#{old_version}...HEAD"
+
+        # Get all commits from previous version (tag) to HEAD
+        resp = Actions::GithubApiAction.run(server_url: 'https://api.github.com',
```

Comment:
I've had to use a lot of actions with this form. I'm not a fan since we don't get the param validation or the default values for these params... But wasn't able to call an action from the helper in any other way. Lmk if you have any recommendations.

Reply:
CC @joshdholtz